openjdk/src/test/java/org/conscrypt/PlatformTest.java

```diff
@@ -177,18 +177,10 @@ public void test_setAndGetSSLParameters() throws Exception {
                 paramsOut.getEndpointIdentificationAlgorithm());
         assertEquals(paramsIn.getWantClientAuth(), paramsOut.getWantClientAuth());
         assertEquals(paramsIn.getNeedClientAuth(), paramsOut.getNeedClientAuth());
-        assertSNIMatchersEqual(paramsIn.getSNIMatchers(), paramsOut.getSNIMatchers());
+        MoreAssertions.assertContentsEqual(paramsIn.getSNIMatchers(), paramsOut.getSNIMatchers());
```

Comment:
assertEquals(sniMatcherSet(paramsIn), sniMatcherSet(paramsOut));

private static<T> Set<SNIMatcher> sniMatcherSet(SSLParameters sslParameters) {
  Collection<SNIMatcher> collection = sslParameters.getSNIMatchers();
  Set<SNIMatcher> result = new HashSet<>(collection);
  return result;
}

or if you also want to forbid duplicates, add:

  assertEquals("Should not have duplicates", result.size(), collection.size());